rlberry/envs/tests/test_bandits.py

```diff
@@ -1,20 +1,35 @@
 import numpy as np
-from rlberry.envs.bandits import NormalBandit, CorruptedNormalBandit
 from rlberry.seeding import safe_reseed
 from rlberry.seeding import Seeder
+from rlberry.envs.bandits import (
+    BernoulliBandit,
+    NormalBandit,
+    CorruptedNormalBandit,
+)
+
+
+TEST_SEED = 42
+
+
+def test_bernoulli():
+    env = BernoulliBandit(p=[0.05, 0.95])
+    safe_reseed(env, Seeder(TEST_SEED))
+
+    sample = [env.step(1)[1] for f in range(1000)]
+    assert np.abs(np.mean(sample) - 0.95) < 0.1
 
 
 def test_normal():
     env = NormalBandit(means=[0, 1])
-    safe_reseed(env, Seeder(42))
+    safe_reseed(env, Seeder(TEST_SEED))
 
     sample = [env.step(1)[1] for f in range(1000)]
     assert np.abs(np.mean(sample) - 1) < 0.1
 
 
 def test_cor_normal():
     env = CorruptedNormalBandit(means=[0, 1], cor_prop=0.1)
-    safe_reseed(env, Seeder(42))
+    safe_reseed(env, Seeder(TEST_SEED))
 
     sample = [env.step(1)[1] for f in range(1000)]
     assert np.abs(np.median(sample) - 1) < 0.5
```

Comment:
Maybe we can also include some generic bandit testing using something like in PR #110 ? You can do it here or I will do a separate PR.

Reply:
Agreed, I'll have a look. Maybe a separate PR for clarity? Ideally it would be great to agree on a good set of tests for bandits and just copy-paste those when we implement more agents, so I think it is worth dedicating some time on this

Reply:
Ok. Ideally, the generic check_bandit function should test only basics properties (reproducibility, save & load and things like that) but maybe we want to keeep some specific tests for each bandit, for instance test that it works well on subgaussian and not only on bernoulli, test that adversarial bandits work on adversarial problems...